engine/runtime/src/bench/java/org/enso/interpreter/bench/benchmarks/semantic/VectorBenchmarks.java

```diff
@@ -73,7 +73,9 @@ public void initializeBenchmark(BenchmarkParams params) throws Exception {
         to_array vec = vec.to_array
         slice vec = vec.slice
         fill_proxy proxy vec =
-          size v = vec.length
+          size v =
```

Comment:
The PR description says:

> Testing Pavel's theory that it is due to logging.

How is the logging related to this change?

Reply:
Because the parameter `v` is unused, and if there is no logger, i.e., `NopLoggingProvider`, no warning is produced, but if there is `org.enso.logger.TestLogProvider`, a warning is produced to the stderr. That is just a theory. Generally, I think that the size of the data for this particular benchmark is too small, the average time of a single iteration is `0.0017` ms.